ui/src/dialogs/AboutDialog.js

```diff
@@ -49,15 +49,29 @@ const AboutDialog = ({ open, onClose }) => {
                   <TableCell align="left"> {config.version} </TableCell>
                 ) : (
                   <TableCell align="left">
-                    <Link
-                      href={`https://github.com/navidrome/navidrome/releases/tag/v${
-                        config.version.split(' ')[0]
-                      }`}
-                      target="_blank"
-                      rel="noopener noreferrer"
-                    >
-                      {config.version.split(' ')[0]}
-                    </Link>
+                    {config.version.includes('SNAPSHOT') ? (
+                      <Link
```

Comment:
This now is too long to be inline. Can you please make it into a function or component?